paasta_tools/remote_git.py

```diff
@@ -81,5 +81,6 @@ def make_force_push_mutate_refs_func(targets, sha):
     def mutate_refs(refs):
         for target in targets:
             refs[target] = sha
+            refs = {k.encode('UTF-8'): v.encode('UTF-8') for k, v in refs.items()}
```

Comment:
If I'm reading it right `make_force_push_mutate_refs_func` is called by `_make_determine_wants_func` but in `create_remote_refs` if `force=True` we only end up calling `make_force_push_mutate_refs_func` so I think it has to go there?